packages/runtime-test/src/nodeOps.ts

```diff
@@ -188,6 +188,28 @@ function remove(child: TestNode, logOp = true) {
   }
 }
 
+function replace(newChild: TestNode, oldChild: TestNode, logOp = true) {
```

Comment:
We should avoid adding more methods to `nodeOps` unless it is absolutely necessary, since custom renderers would need to implement all these methods, and if a new method is required for the renderer to work then it is technically a breaking change for previously implemented custom renderers.

Reply:
done.
BTW, the CI error related to https://github.com/vuejs/core/commit/733437691f70ebca8dd6cc3bc8356f5b57d4d5d8